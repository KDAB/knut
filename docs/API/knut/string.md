# String

Description of a RC file string. [More...](#detailed-description)

```qml
import Knut
```

## Properties

| | Name |
|-|-|
|string|**[id](#id)**|
|string|**[text](#text)**|

## Property Documentation

#### <a name="id"></a>string **id**

This property holds the id of the string.

#### <a name="text"></a>string **text**

This property holds the text of the string.
