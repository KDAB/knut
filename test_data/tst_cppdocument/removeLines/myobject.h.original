#pragma once

#include <string>

class MyObject {
public:
    MyObject(const std::string& message);
    // this comment should stay

    ~MyObject();// this comment should be removed

    // this comment should be removed
    void sayMessage();

    /* this comment should be removed */
    void sayMessage(const std::string& test);

    // this comment should be removed
    // this one too
    void sayMessage(const std::string& test, int num);

private:
    std::string m_message;
};
