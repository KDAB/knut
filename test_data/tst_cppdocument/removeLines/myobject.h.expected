#pragma once

#include <string>

class MyObject {
public:
    MyObject(const std::string& message);
    // this comment should stay

private:
    std::string m_message;
};
